app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -888,6 +887,23 @@ impl Node {
                 <+ visualization_visible.not().and(&no_error_set);
 
 
+            // === Bounding Box ===
+
+            let visualization_size             = &model.visualization.frp.size;
+            visualization_enabled_and_visible <- visualization_enabled && visualization_visible;
+            bounding_box_input                <- all4(
```

Comment:
What does `bounding_box_input` mean?

Reply:
As we [discussed](https://discord.com/channels/401396655599124480/949323065320415292/949332133476966460), I'm leaving the name, only shortening it to `bbox_input`.